datafusion/substrait/src/logical_plan/producer.rs

```diff
@@ -80,7 +83,7 @@ pub fn to_substrait_plan(plan: &LogicalPlan) -> Result<Box<Plan>> {
 
     // Return parsed plan
     Ok(Box::new(Plan {
-        version: None, // TODO: https://github.com/apache/arrow-datafusion/issues/4949
+        version: Some(version::version_with_producer("datafusion")),
```

Comment:
We have both logical plan and physical plan. Is it needed to distinguish them in this version field?

Reply:
No, that is not something that should be encoded in the version field of a plan.